.github/workflos/main.yml

```diff
@@ -0,0 +1,32 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
The directory name is wrong (workflos vs workflows)

Reply:
Thanks! Resolved in 2fb8a9f